packages/salesforcedx-apex-debugger/test/unit/core/streamingService.test.ts

```diff
@@ -219,7 +218,7 @@ describe('Debugger streaming service', () => {
       );
     });
 
-    after(() => {
+    afterAll(() => {
```

Comment:
one difference between mocha and jest is jest has an afterAll instead of just after.